domains/door.json

```diff
@@ -1,10 +1,10 @@
 {
   "description": "A world of doors",
   "owner": {
-    "username": "ukrioo",
-    "email": "door@ukrio.is-a.dev"
+    "username": "ukriu",
+    "email": "door-isadev@ukriu.com"
   },
   "record": {
-    "CNAME": "ukrioo.github.io"
+    "URL": "https://door.ukriu.com/"
```

Comment:
I cannot reach your website
`DNS_PROBE_FINISHED_NXDOMAIN`

Reply:
Deleted `door` as well.
(Quite frankly it was just a little door animation which switches between door a seemingly infinite amount of doors, its useless)